EpiAware/src/inference-methods.jl

```diff
@@ -0,0 +1,97 @@
+"""
+Run pathfinder multiple times and store the results in an array. Fails safely.
+
+# Arguments
+- `mdl::DynamicPPL.Model`: The `Turing` model to be used for inference.
+- `nruns`: The number of times to run the `pathfinder` function.
+- `kwargs...`: Additional keyword arguments passed to `pathfinder`.
+
+# Returns
+An array of `PathfinderResult` objects or `Symbol` values indicating success or failure.
+"""
+function _run_manypathfinder(mdl::DynamicPPL.Model; nruns, kwargs...)
+    @info "Running pathfinder $nruns times"
+    pfs = Vector{Union{Pathfinder.PathfinderResult, Symbol}}(undef, nruns)
+    Threads.@threads for i in 1:nruns
```

Comment:
is threads built in (hence not needing to import it?)

Reply:
Yes.